### PR TITLE
[PD-10921] To revoke a users access token when the app is being uninstalled

### PR DESCRIPTION
**Pull request by TamasSzilagyi84**
### Purpose:

To revoke the user's token when the application is being disconnected.

### What Changed:

A new resource has been added, called `Token`, with one method to allow us to revoke the user's token.

`OAuthTokenHandler` has been updated to override the authorization header during token revocation and to append the access token to the Uri

### Steps to test

1. Connect with a BYO license
2. Disconnect your BYO license

### CODE REVIEW

File: Source/ZoomNet/Resources/IToken.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	/// Allows you to manage tokens.
+	/// </summary>
+	public interface IToken
+	{
+		/// <summary>
+		/// To revoke a token.
+		/// </summary>
+		/// <param name="cancellationToken">Cancellation token.</param>
+		/// <returns>TokenRevocation object.</returns>
+		Task<TokenRevocation> RevokeAsync(CancellationToken cancellationToken = default);
```
Comment (MaiklSemenov) [inline Source/ZoomNet/Resources/IToken.cs]: Do you think we can simplify it and just move this method to the `ZoomOAuthClient ` class?

Comment (TamasSzilagyi84) [reply Source/ZoomNet/Resources/IToken.cs]: `Token` resource has been removed and the `RevokeAsnyc` method moved inside the client

File: Source/ZoomNet/ZoomOAuthClient.cs [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+		/// <summary>
+		/// Revokes the access token of the user.
+		/// </summary>
+		/// <param name="cancellationToken">CancellationToken.</param>
+		/// <returns>TokenRevocation object.</returns>
+		public Task<TokenRevocation> RevokeAsync(CancellationToken cancellationToken)
+		{
+			return _fluentClient
+				.PostAsync("https://zoom.us/oauth/revoke")
```
Comment (MaiklSemenov) [inline Source/ZoomNet/ZoomOAuthClient.cs]: it looks like we don't need to provide `https://zoom.us/oauth/` here, right?

Comment (TamasSzilagyi84) [reply Source/ZoomNet/ZoomOAuthClient.cs]: That is correct and has been removed

**Review by MaiklSemenov — APPROVED** [review]
It looks good to me 👍

**Pull request merged**